src/EventStore.Core/GossipStatsCollector.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Diagnostics.Metrics;
+using EventStore.Core.Telemetry;
+
+namespace EventStore.Core;
+
+public class GossipStatsCollector {
+	private readonly DurationTracker _tracker;
+
+	internal GossipStatsCollector(DurationMetric metric, string name) {
+		_tracker = new DurationTracker(metric, name);
+	}
+
+	public IDisposable StartRequest() {
+		return _tracker.Start();
+	}
+}
+
+public class GossipStatsCollectors {
+	public static readonly GossipStatsCollectors Default = new GossipStatsCollectors();
+	private Meter _coreMeter;
+	private DurationMetric _metric;
+
+	public Meter CoreMeter {
+		set {
+			_coreMeter = value;
```

Comment:
do we need the meter in here, it may be sufficient to just have hte metric, and to pass it in in the constructor